resources/checks/A006_pg_conf.sh

```diff
@@ -0,0 +1,7 @@
+# Collect pg_settings artifact
```

Comment:
Please improve a header comment. For understanding 'what happening' without reading a whole code down to the last line.